packages/wrangler/src/pages/functions/buildWorker.ts

```diff
@@ -339,7 +339,7 @@ function blockWorkerJsImports(nodejsCompatMode: NodeJSCompatMode): Plugin {
 	return {
 		name: "block-worker-js-imports",
 		setup(build) {
-			build.onResolve({ filter: /.*/g }, (args) => {
+			build.onResolve({ filter: /.*/ }, (args) => {
```

Comment:
[esbuild v0.25.2](https://github.com/evanw/esbuild/releases/tag/v0.25.2) has changed how it translate it into Go regular expressions, which I believes the `g` flag was unnecessary 🤔 Let me know if you disagree